ci/long_running_tests/README.rst

```diff
@@ -1,29 +1,36 @@
 Long Running Tests
 ==================
 
-This directory contains scripts for starting long-running workloads which are
-intended to run forever until they fail.
+This directory contains the long-running workloads which are intended to run
+forever until they fail. To set up the project you need to run
+
+.. code-block:: bash
+
+    pip install any
+    any project create
```

Comment:
`ray project create`?

Reply:
This actually is needed to register it with the project database in the `any` command.